<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html>
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
  <meta http-equiv="Content-Style-Type" content="text/css">
  <title></title>
  <meta name="Generator" content="Cocoa HTML Writer">
  <meta name="CocoaVersion" content="1671.6">
  <style type="text/css">
    p.p1 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 12.0px Helvetica; color: #000000; -webkit-text-stroke: #000000}
    p.p2 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 12.0px Helvetica; color: #000000; -webkit-text-stroke: #000000; min-height: 12.0px}
    p.p3 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 12.0px Helvetica; color: #000000}
    p.p4 {margin: 0.0px 0.0px 0.0px 0.0px; line-height: 14.0px; font: 12.0px Helvetica; color: #000000; -webkit-text-stroke: #000000; min-height: 14.0px}
    span.s1 {font-kerning: none}
    span.s2 {font-kerning: none; -webkit-text-stroke: 0px #000000}
  </style>
</head>
<body>
<p class="p1"><span class="s1">&lt;script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"&gt;&lt;/script&gt;</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"&gt;&lt;/script&gt;</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;script type="text/javascript" src="//cdn.plu.mx/widget-popup.js"&gt;&lt;/script&gt;</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;script type="text/javascript" src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"&gt;&lt;/script&gt;</span></p>
<p class="p1"><span class="s1">&lt;h1&gt;PUBLICATIONS&lt;/h1&gt;</span></p>
<p class="p1"><span class="s1">New and related publications are presented on the &lt;a href="http://openboxscience.org"&gt;Open Box Science&lt;/a&gt; platform for scientists around the world to freely hear and discuss!</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;ISMMS Lab Members | &lt;span style="color: #800080"&gt;Volunteer/Collaboration Team Members &lt;span style="color: #333333"&gt;| *Co-first Authors | #Co-corresponding Authors | "Consortium authorship&lt;/span&gt;&lt;/span&gt;&lt;/strong&gt;</span></p>
<p class="p1"><span class="s1">&lt;h3&gt;&lt;/h3&gt;<br>
</span></p>
<p class="p1"><span class="s1">&amp;nbsp;</span></p>
<p class="p1"><span class="s1">&lt;h3&gt;2021&lt;/h3&gt;</span></p>
<p class="p2"><span class="s1"></span><br></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/s42003-021-02636-7" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/s42003-021-02636-7"&gt;Pan-cancer Proteogenomic Investigations Identify Post-Transcriptional Kinase Targets&lt;/a&gt;</span></p>
<p class="p3"><span class="s2">&lt;strong&gt;</span><span class="s1">Elmas A</span><span class="s2">&lt;/strong&gt;, </span><span class="s1">&lt;strong&gt;Tharakan S&lt;/strong&gt;, &lt;strong&gt;Jaladanki S&lt;/strong&gt;,<span class="Apple-converted-space">  </span>Galsky M, Liu T, </span><span class="s2">&lt;strong&gt;Huang KL&lt;/strong&gt;</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Communications Biology&lt;/em&gt; 2021. [&lt;a href="https://github.com/Huang-lab/oppti"&gt;GitHub&lt;/a&gt;]</span></p>
<p class="p4"><span class="s1"></span><br></p>
<p class="p3"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.3390/cancers13184572" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p3"><span class="s1">&lt;a href="https://www.mdpi.com/2072-6694/13/18/4572"&gt;Genomic Determinants of Homologous Recombination Deficiency across Human Cancers&lt;/a&gt;</span></p>
<p class="p3"><span class="s1">&lt;span style="color: #800080"&gt;&lt;strong&gt;Qing T&lt;/strong&gt;&lt;/span&gt;, &lt;span style="color: #800080"&gt;&lt;strong&gt;X Wang&lt;/strong&gt;&lt;/span&gt;, &lt;strong&gt;Jun T&lt;/strong&gt;, Ding L, &lt;strong&gt;Pusztai L#&lt;/strong&gt;, &lt;strong&gt;Huang KL#&lt;/strong&gt;.</span></p>
<p class="p3"><span class="s1">&lt;em&gt;Cancers&lt;/em&gt; 2021.<span class="Apple-converted-space"> </span></span></p>
<p class="p4"><span class="s1"></span><br></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1186/s13073-021-00964-1" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-021-00964-1"&gt;Non-cancer-related pathogenic germline variants and expression consequences in ten-thousand cancer genomes&lt;/a&gt;</span></p>
<p class="p1"><span class="s2">&lt;strong&gt;Wang Z&lt;/strong&gt;</span><span class="s1">, Fan X, Shen Y, Pagadala MS, </span><span class="s2">&lt;strong&gt;</span><span class="s1">Signer R</span><span class="s2">&lt;/strong&gt;</span><span class="s1">, Cygan KJ, Fairbrother WG, Carter H, &lt;strong&gt;</span><span class="s2">Chung WK</span><span class="s1">#&lt;/strong&gt;, &lt;strong&gt;Huang KL#</span><span class="s2">&lt;/strong&gt;.</span></p>
<p class="p3"><span class="s2">&lt;em&gt;Genome Medicine&lt;/em&gt; 2021. </span><span class="s1">[&lt;a href="https://github.com/WangZishan/TCGASecondaryPredisposition"&gt;GitHub&lt;/a&gt;]</span></p>
<p class="p2"><span class="s1"></span><br></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1016/j.xpro.2021.100766" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://linkinghub.elsevier.com/retrieve/pii/S2666-1667(21)00472-X"&gt;Analytical protocol to identify local ancestry-associated molecular features in cancer&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Carrot-Zhang J, Han S, Zhou W, Damrauer JS, Kemal A; &lt;strong&gt;Cancer Genome Atlas Analysis Network”&lt;/strong&gt;, Cherniack AD, Beroukhim R.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;STAR Protoc.&lt;/em&gt;&amp;nbsp;2021.<br>
</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi=</span><span class="s2">"</span><span class="s1">doi.org/10.1038/s41598-021-93126-7" </span><span class="s2">data-hide-no-mentions="true" </span><span class="s1">class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/s41598-021-93126-7"&gt;Prediction of individual COVID-19 diagnosis using baseline demographics and lab data&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;&lt;span style="color: #800080;"&gt;Zhang J&lt;/span&gt;, Jun T&lt;/strong&gt;, &lt;strong&gt;&lt;span style="color: #800080;"&gt;Frank J&lt;/span&gt;&lt;/strong&gt;, Nirenberg S, Kovatch P, &lt;strong&gt;Huang KL&lt;/strong&gt;.</span></p>
<p class="p3"><span class="s2">&lt;em&gt;Scientific Reports&lt;/em&gt; 2021. </span><span class="s1">[&lt;a href="https://github.com/Huang-lab/covid19-diagnosis-prediction"&gt;GitHub&lt;/a&gt;]</span><span class="s2"><br>
</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi=</span><span class="s2">"</span><span class="s1">doi.org/10.1038/s43856-021-00006-2" </span><span class="s2">data-hide-no-mentions="true" </span><span class="s1">class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/s43856-021-00006-2.epdf"&gt;Analysis of sex-speciﬁc risk factors and clinical outcomes in COVID-19&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Jun T&lt;/strong&gt;, Nirenberg S, &lt;strong&gt;Weinberger T&lt;/strong&gt;, &lt;span style="color: #800080"&gt;&lt;strong&gt;Sharma N&lt;/strong&gt;&lt;/span&gt;, Pujadas E, Cordon-Cardo C, Kovatch P, &lt;strong&gt;Huang KL&lt;/strong&gt;.<span class="Apple-converted-space"> </span></span></p>
<p class="p1"><span class="s1">&lt;em&gt;Communications Medicine&lt;/em&gt; 2021.<br>
</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.2147/CMAR.S311548" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.dovepress.com/the-functional-hallmarks-of-cancer-predisposition-genes-peer-reviewed-fulltext-article-CMAR"&gt;The Functional Hallmarks of Cancer Predisposition Genes&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Capellini A*&lt;/strong&gt;, &lt;strong&gt;Williams M*&lt;/strong&gt;, Onel K#, &lt;strong&gt;Huang KL#&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Cancer Manag Res.&lt;/em&gt;&amp;nbsp;2021.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/s41598-021-91713-2" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/s41598-021-91713-2"&gt;Genetic dependency of Alzheimer’s disease-associated genes across cells and tissue types&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Jaladanki SK*&lt;/strong&gt;, &lt;strong&gt;Elmas A*&lt;/strong&gt;, &lt;strong&gt;Malave GS&lt;/strong&gt;, &lt;strong&gt;Huang KL&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Scientific Reports&lt;/em&gt; 2021.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1016/j.isci.2021.102550" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.sciencedirect.com/science/article/pii/S2589004221005186"&gt;Phenome-wide and expression quantitative trait locus associations of COVID-19 genetic risk loci&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Moon C&lt;/strong&gt;, Schilder BM, Raj T, &lt;strong&gt;Huang KL&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;iScience&lt;/em&gt; 2021.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1016/j.xcrm.2021.100276" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.sciencedirect.com/science/article/pii/S266637912100104X"&gt;Diverse Immune Response of DNA Damage Repair-Deficient Tumors&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;span style="color: #800080"&gt;&lt;strong&gt;Qing T&lt;/strong&gt;&lt;/span&gt;, &lt;strong&gt;Jun T&lt;/strong&gt;, Lindblad KE, Lujambio A, Marczyk M, Pusztai L#, &lt;strong&gt;Huang KL#&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Cell Reports Medicine&lt;/em&gt; 2021.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/s41467-021-22481-w" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/s41467-021-22481-w"&gt;Systematic discovery of spatially interacting phosphorylation sites and mutations in cancer&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Huang KL#&lt;/strong&gt;, Scott AD, Zhou DC, Wang LB, Weerasinghe A, &lt;strong&gt;Elmas A&lt;/strong&gt;, Liu R, Sengupta S, Wu Y, Wendl MC, Wyczalkowski MA, Baral J, Sengupta S, Lai CW, Ruggles K, Payne SH, Raphael B, Fenyö D, Chen K, Mills G, Ding L#.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature Communications&lt;/em&gt; 2021. [&lt;a href="https://github.com/ding-lab/hotspot3d"&gt;GitHub for software&lt;/a&gt;][&lt;a href="https://github.com/ding-lab/HotPho_Analysis"&gt;GitHub for analysis scripts&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1142/9789811232701_0017" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.worldscientific.com/doi/abs/10.1142/9789811232701_0017"&gt;AeQTL: eQTL analysis using region-based aggregation of rare variants&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;span style="color: #800080"&gt;&lt;strong&gt;Dong G&lt;/strong&gt;&lt;/span&gt;, Wendl M, Zhang B, Ding L,&lt;strong&gt; Huang KL&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Pac Symp Biocomput.&lt;/em&gt; 2021. &lt;em&gt;[&lt;a href="https://github.com/Huang-lab/AeQTL"&gt;GitHub&lt;/a&gt;]&lt;/em&gt;</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;h3&gt;2020&lt;/h3&gt;</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1016/j.cell.2020.11.004" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.sciencedirect.com/science/article/pii/S009286742031463X"&gt;Shared Immunogenic Poly-Epitope Frameshift Mutations in Microsatellite Unstable Tumors&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Roudko V, Bozkus CC, Orfanelli T, McClain CB, Carr C, O'Donnell T, Chakraborty L, Samstein R, &lt;strong&gt;Huang KL&lt;/strong&gt;, Blank SV, Greenbaum B, Bhardwaj N.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Cell&lt;/em&gt;&amp;nbsp;2020.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/s41467-020-18534-1" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://pubmed.ncbi.nlm.nih.gov/32968074/"&gt;Risk prediction of late-onset Alzheimer's disease implies an oligogenic architecture&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Zhang Q, Sidorenko J, Couvy-Duchesne B, Marioni RE, Wright MJ, Goate AM, Marcora E,&lt;b&gt;&amp;nbsp;Huang KL&lt;/b&gt;, Porter T, Laws SM; Australian Imaging Biomarkers and Lifestyle (AIBL) Study, Sachdev PS, Mather KA, Armstrong NJ, Thalamuthu A, Brodaty H, Yengo L, Yang J, Wray NR, McRae AF, Visscher PM.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature Communications&lt;/em&gt;&amp;nbsp;2020.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/s41467-020-18151-y" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/s41467-020-18151-y"&gt;Retrospective evaluation of whole exome and genome mutation calls in 746 cancer samples.&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Bailey MH, Meyerson WU, Dursi LJ, Wang LB, Dong G, Liang WW, Weerasinghe A, Li S, Kelso S; MC3 Working Group; PCAWG novel somatic mutation calling methods working group, Saksena G, Ellrott K, Wendl MC, Wheeler DA, Getz G, Simpson JT, Gerstein MB, Ding L;&amp;nbsp;&lt;strong&gt;PCAWG Consortium"&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature Communications&lt;/em&gt;&amp;nbsp;2020.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/s41467-020-17359-2" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/s41467-020-17359-2"&gt;Sex differences in oncogenic mutational processes. &lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Li CH, Prokopec SD, Sun RX, Yousif F, Schmitz N, Boutros PC; &lt;strong&gt;PCAWG Consortium"&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature Communications&lt;/em&gt;&amp;nbsp;2020.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1053/j.gastro.2020.08.015" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.sciencedirect.com/science/article/pii/S001650852035054X"&gt;Cooperation between distinct cancer driver genes underlies inter-tumor heterogeneity in hepatocellular carcinoma&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Molina-Sánchez P, Ruiz de Galarreta M, Yao MA, Lindblad KE, Bresnahan E, Bitterman E, Martin TC, Rubenstein T, Nie K, Golas J, Choudhary S, Bárcena-Varela M, &lt;strong&gt;Elmas A&lt;/strong&gt;, Miguela V, Ding Y, Kan Z, Grinspan LT,&lt;b&gt;&amp;nbsp;Huang KL&lt;/b&gt;, Parsons RE, Shields DJ, Rollins RA, Lujambio A.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Gastroenterology&lt;/em&gt;&amp;nbsp;2020.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1186/s13073-020-00744-3" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-020-00744-3"&gt;Ancestry-Specific Predisposing Germline Variants in Cancer&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;span class="highwire-citation-author first"&gt;&lt;span class="nlm-surname"&gt;&lt;span style="color: #800080"&gt;&lt;strong&gt;Oak N&lt;/strong&gt;&lt;/span&gt;&lt;/span&gt;&lt;/span&gt;,&amp;nbsp;Cherniack AD, Mashl RJ; TCGA Analysis Network, Hirsch FR, Ding L, Beroukhim R, Gümüş ZH, Plon SE, &lt;strong&gt;Huang KL.&lt;/strong&gt;</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Genome Medicine&lt;/em&gt; 2020. [&lt;a href="https://www.biorxiv.org/content/10.1101/2020.04.14.032557v1"&gt;bioRxiv&lt;/a&gt;][&lt;a href="https://github.com/Huang-lab/AncestralCancerPredisposition"&gt;GitHub&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1111/ahg.12389" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://onlinelibrary.wiley.com/doi/abs/10.1111/ahg.12389"&gt;Genotype Concordance and Polygenic Risk Score Estimation across Consumer Genetic Testing Data&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;span style="color: #800080"&gt;&lt;strong&gt;Batra P&lt;/strong&gt;&lt;/span&gt;, &lt;strong&gt;Huang KL.&lt;/strong&gt;</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Annals of Human Genetics&lt;/em&gt; 2020. [&lt;a href="https://www.biorxiv.org/content/10.1101/795666v2"&gt;bioRxiv&lt;/a&gt;][&lt;a href="https://github.com/Huang-lab/AncestralCancerPredisposition"&gt;GitHub&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi=</span><span class="s2">"</span><span class="s1">doi.org/10.1016/j.ccell.2020.04.012" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.cell.com/cancer-cell/pdf/S1535-6108(20)30211-7.pdf"&gt;Comprehensive Analysis of Genetic Ancestry and Its Molecular Correlates in Cancer&amp;nbsp;&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Carrot-Zhang J*, Chambwe N*, Damrauer JS*, Knijnenburg TA*, Robertson AG*, Yau C*, Zhou W*, Berger AC*, &lt;strong&gt;Huang KL&lt;/strong&gt;*, Newberg JY*, Mashl RJ, Romanel A, Sayaman RW, Demichelis F, Felau I, Frampton GM, Han S, Hoadley KA, Kemal A, Laird PW, Lazar AJ, Le X, Oak N, Shen H, Wong CK, Zenklusen JC, Ziv E, Cancer Genome Atlas Analysis Network, Cherniack AD, Beroukhim R.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Cancer Cell &lt;/em&gt;2020.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1371/journal.pcbi.1007723" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1007723"&gt;Ten Simple Rules for landing on the right job after your PhD or postdoc&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Huang KL&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;PLoS Computational Biology&lt;/em&gt; 2020. [&lt;a href="https://www.youtube.com/watch?v=O6HZJgqhxA4"&gt;YouTube&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/s41586-020-1969-6" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://doi.org/10.1038/s41586-020-1969-6"&gt;Pan-cancer analysis of whole genomes&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;ICGC/TCGA Pan-Cancer Analysis of Whole Genomes Consortium"&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature&lt;/em&gt; 2020.</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;h3&gt;2019&lt;/h3&gt;</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1074/mcp.RA118.001243" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.sciencedirect.com/science/article/pii/S1535947620340639"&gt;Regulated phosphosignaling associated with breast cancer subtypes and druggability&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Huang KL&lt;span style="color: #800080"&gt;&lt;span style="color: #333333"&gt;#&lt;/span&gt;&lt;/span&gt;&lt;/strong&gt;, Wu Y, McMichael JF, Scott AD, Cao S, Wendl MC, Johnson KJ, Ruggles K, Held J, Payne SH, Davies S, Ellis MJ, Fenyö D, Chen F, Townsend RR, Carr SA, Ding L&lt;span style="color: #800080"&gt;&lt;span style="color: #333333"&gt;#.&lt;/span&gt;&lt;/span&gt;</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Molecular and Cellular Proteomics&lt;/em&gt; 2019. [&lt;a href="https://github.com/ding-lab/phosphoproteomics"&gt;GitHub&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1002/hep.30677" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://aasldpubs.onlinelibrary.wiley.com/doi/pdf/10.1002/hep.30677"&gt;Mannose Phosphate Isomerase and Mannose Regulate Hepatic Stellate Cell Activation and Fibrosis in Zebrafish and Humans&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">DeRossi C, Bambino K, Morrison J, Sakarin I, Villacorta-Martin C, Zhang C, Ellis JL, Fiel MI, Ybanez M, Lee YA, &lt;strong&gt;Huang KL&lt;/strong&gt;, Yin C, Sakaguchi TF, Friedman SL, Villanueva A, Chu J.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Hepatology&lt;/em&gt; 2019.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1371/journal.pgen.1008049" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1008049"&gt;Functional analysis of BARD1 missense variants in homology-directed repair and damage sensitivity&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Adamovich AI, Banerjee T, Wingo M, Duncan K, Ning J, Martins Rodrigues F, &lt;strong&gt;Huang KL&lt;/strong&gt;, Lee C, Chen F, Ding L, Parvin JD.</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;em&gt;PLoS Genetics&lt;/em&gt; 2019.</span></p>
<p class="p1"><span class="s1">&lt;h3&gt;2018&lt;/h3&gt;</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1002/humu.23668" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/humu.23668"&gt;Framework for microRNA variant annotation and prioritization using human population and disease datasets&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Oak N, Ghosh R, &lt;strong&gt;Huang KL&lt;/strong&gt;, Wheeler DA, Ding L, Plon SE.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Human Mutation&lt;/em&gt;. 2018 Oct 10. [&lt;a href="https://github.com/nroak/ADmiRE"&gt;GitHub&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1093/bioinformatics/bty649" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://academic.oup.com/bioinformatics/advance-article-pdf/doi/10.1093/bioinformatics/bty649/25542157/bty649.pdf"&gt;CharGer: Clinical Characterization of Germline Variants&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Scott AD#, &lt;strong&gt;Huang KL&lt;/strong&gt;, Weerasinghe A, Mashl RJ, Gao Q, Martins Rodrigues F, Wyczalkowski M, Ding L#.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Bioinformatics&lt;/em&gt;. 2018 Aug 9.&amp;nbsp;[&lt;a href="https://github.com/ding-lab/CharGer"&gt;GitHub&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1186/s13073-018-0564-z" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://genomemedicine.biomedcentral.com/articles/10.1186/s13073-018-0564-z"&gt;Integrative omics analyses broaden treatment targets in human cancer&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Sengupta S*, Sun SQ*, &lt;strong&gt;Huang KL&lt;/strong&gt;, Oh C, Bailey MH, Varghese R, Wyczalkowski MA, Ning J, Tripathi P, McMichael JF, Johnson KJ, Kandoth C, Welch J, Ma C, Wendl MC, Payne SH, Fenyö D, Townsend RR, Dipersio JF, Chen F, Ding L.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Genome Medicine&lt;/em&gt;. 2018 Jul 27;10(1):60.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1016/j.cell.2018.03.039" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.cell.com/action/showPdf?pii=S0092-8674%2818%2930363-5"&gt;Pathogenic Germline Variants in 10,389 Adult Cancers&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt; Huang KL&lt;/strong&gt;, Mashl RJ, Wu Y, Ritter DI, Wang J, Oh C, Paczkowska M, Reynolds S, Wyczalkowski MA, Oak N, Scott AD, Krassowski M, Cherniack AD, Houlahan KE, Jayasinghe R, Wang LB, Zhou DC, Liu D, Cao S, Kim YW, Koire A, McMichael JF, Hucthagowder V, Kim TB, Hahn A, Wang C, McLellan MD, Al-Mulla F, Johnson KJ; Cancer Genome Atlas Research Network, Lichtarge O, Boutros PC, Raphael B, Lazar AJ, Zhang W, Wendl MC, Govindan R, Jain S, Wheeler D, Kulkarni S, Dipersio JF, Reimand J, Meric-Bernstam F, Chen K, Shmulevich I, Plon SE, Chen F#, Ding L#.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Cell&lt;/em&gt;. 2018 Apr 5;173(2):355-370.e14. [&lt;a href="https://github.com/ding-lab/PanCanAtlasGermline"&gt;GitHub&lt;/a&gt;] [Controlled-access variant-call data available: &lt;a href="https://gdc.cancer.gov/about-data/publications/PanCanAtlas-Germline-AWG"&gt;GDC&lt;/a&gt;] [Presentation: &lt;a href="https://www.youtube.com/watch?v=-jDVm8_x-zM&amp;amp;t=6s"&gt;YouTube&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1016/j.cell.2018.03.033" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.sciencedirect.com/science/article/pii/S0092867418303131"&gt;Perspective on Oncogenic Processes at the End of the Beginning of Cancer Genomics&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Ding L#, Bailey MH, Porta-Pardo E, Thorsson V, Colaprico A, Bertrand D, Gibbs DL, Weerasinghe A, &lt;strong&gt;Huang KL&lt;/strong&gt;, Tokheim C, Cortés-Ciriano I, Jayasinghe R, Chen F, Yu L, Sun S, Olsen C, Kim J, Taylor AM, Cherniack AD, Akbani R, Suphavilai C, Nagarajan N, Stuart JM, Mills GB, Wyczalkowski MA, Vincent BG, Hutter CM, Zenklusen JC, Hoadley KA, Wendl MC, Shmulevich L, Lazar AJ, Wheeler DA#, Getz G#; Cancer Genome Atlas Research Network.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Cell&lt;/em&gt;. 2018 Apr 5;173(2):305-320.e10.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/s41422-018-0019-5" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/s41422-018-0019-5"&gt;Pan-cancer analysis of somatic mutations across 21 neuroendocrine tumor types&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Cao Y, Zhou W, Li L, Wang J, Gao Z, Jiang Y, Jiang X, Shan A, Bailey MH, &lt;strong&gt;Huang&lt;/strong&gt; &lt;strong&gt;KL&lt;/strong&gt;, Sun SQ, McLellan MD, Niu B, Wang W, Ding L, Ning G.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Cell Research&lt;/em&gt;. 2018 May;28(5):601-604.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1158/0008-5472.CAN-17-1990" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="http://cancerres.aacrjournals.org/content/78/10/2732"&gt;Mass Spectrometry-Based Proteomics Reveals Potential Roles of NEK9 and MAP2K4 in Resistance to PI3K Inhibition in Triple-Negative Breast Cancers&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Mundt F, Rajput S, Li S, Ruggles KV, Mooradian AD, Mertins P, Gillette MA, Krug K, Guo Z, Hoog J, Erdmann-Gilmore P, Primeau T, Huang S, Edwards DP, Wang X, Wang X, Kawaler E, Mani DR, Clauser KR, Gao F, Luo J, Davies SR, Johnson GL, &lt;strong&gt;Huang KL&lt;/strong&gt;, Yoon CJ, Ding L, Fenyö D, Ellis MJ, Townsend RR, Held JM, Carr SA, Ma CX.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Cancer Research&lt;/em&gt;. 2018 May 15;78(10):2732-2746.</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/d41586-018-01076-4" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/d41586-018-01076-4"&gt;Most popular public searches on gene names&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Huang KL&lt;/strong&gt;.</span></p>
<p class="p1"><span class="s1">&lt;em&gt; Nature&lt;/em&gt;. 2018 Jan 25;553(7689):405. [&lt;a href="go.nature.com/2dsjvdm"&gt;Gene Google Trend Data&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;h3&gt;2017&lt;/h3&gt;</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1126/scisignal.aam8065" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="http://stke.sciencemag.org/content/10/491/eaam8065"&gt;Breast tumors educate the proteome of stromal tissue in an individualized but coordinated manner&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Wang X, Mooradian AD, Erdmann-Gilmore P, Zhang Q, Viner R, Davies SR,&lt;strong&gt; Huang&lt;/strong&gt; &lt;strong&gt;KL&lt;/strong&gt;, Bomgarden R, Van Tine BA, Shao J, Ding L, Li S, Ellis MJ, Rogers JC, Townsend RR, Fenyö D, Held JM.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Science Signalling&lt;/em&gt;. 2017 Aug 8;10(491).</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/nn.4587" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/nn.4587"&gt;A common haplotype lowers PU.1 expression in myeloid cells and delays onset of Alzheimer's disease&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Huang KL*&lt;/strong&gt;, Marcora E*, Pimenova AA, Di Narzo AF, Kapoor M, Jin SC, Harari O, Bertelsen S, Fairfax BP, Czajkowski J, Chouraki V, Grenier-Boley B, Bellenguez C, Deming Y, McKenzie A, Raj T, Renton AE, Budde J, Smith A, Fitzpatrick A, Bis JC, DeStefano A, Adams HHH, Ikram MA, van der Lee S, Del-Aguila JL, Fernandez MV, Ibañez L; International Genomics of Alzheimer's Project; Alzheimer's Disease Neuroimaging Initiative, Sims R, Escott-Price V, Mayeux R, Haines JL, Farrer LA, Pericak-Vance MA, Lambert JC, van Duijn C, Launer L, Seshadri S, Williams J, Amouyel P, Schellenberg GD, Zhang B, Borecki I, Kauwe JSK, Cruchaga C, Hao K, Goate AM.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature Neuroscience&lt;/em&gt;. 2017 Aug;20(8):1052-1061. [&lt;a href="https://github.com/kuanlinhuang/AD_SPI1_project"&gt;GitHub&lt;/a&gt;][&lt;a href="https://www.youtube.com/watch?v=3XiOkwo0X_4&amp;amp;t=2s"&gt;YouTube&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1101/gr.211656.116" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://genome.cshlp.org/content/27/8/1450.full"&gt;GenomeVIP: a cloud platform for genomic variant discovery and interpretation&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Mashl RJ, Scott AD, &lt;strong&gt;Huang KL&lt;/strong&gt;, Wyczalkowski MA, Yoon CJ, Niu B, DeNardo E, Yellapantula VD, Handsaker RE, Chen K, Koboldt DC, Ye K, Fenyö D, Raphael BJ, Wendl MC, Ding L.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Genome Research&lt;/em&gt;. 2017 Aug;27(8):1450-1459.&amp;nbsp;[&lt;a href="https://github.com/ding-lab/GenomeVIP"&gt;GitHub&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/ncomms14864" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/ncomms14864/"&gt;Proteogenomic integration reveals therapeutic targets in breast cancer xenografts&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">&lt;strong&gt;Huang KL&lt;/strong&gt;*, Li S*, Mertins P*, Cao S, Gunawardena HP, Ruggles KV, Mani DR, Clauser KR, Tanioka M, Usary J, Kavuri SM, Xie L, Yoon C, Qiao JW, Wrobel J, Wyczalkowski MA, Erdmann-Gilmore P, Snider JE, Hoog J, Singh P, Niu B, Guo Z, Sun SQ, Sanati S, Kawaler E, Wang X, Scott A, Ye K, McLellan MD, Wendl MC, Malovannaya A, Held JM, Gillette MA, Fenyö D, Kinsinger CR, Mesri M, Rodriguez H, Davies SR, Perou CM, Ma C, Townsend RR, Chen X, Carr SA#, Ellis MJ#, Ding L#.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature Communications&lt;/em&gt;. 2017 Mar 28;8:14864. [&lt;a href="https://github.com/kuanlinhuang/PDXNatComm2017"&gt;GitHub&lt;/a&gt;][&lt;a href="https://mountsinai.box.com/s/3gzbcvizzfy9oev54yarysqxjwl6r47o"&gt;Shared multi-omic data for the PDX cohort&lt;/a&gt;][&lt;a href="https://www.youtube.com/watch?v=c3QaJ19sjS4"&gt;YouTube&lt;/a&gt;]</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1007/s00401-017-1685-y" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://link.springer.com/article/10.1007%2Fs00401-017-1685-y"&gt;Genome-wide association study identifies four novel loci associated with Alzheimer's endophenotypes and disease modifiers&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Deming Y, Li Z, Kapoor M, Harari O, Del-Aguila JL, Black K, Carrell D, Cai Y, Fernandez MV, Budde J, Ma S, Saef B, Howells B, &lt;strong&gt;Huang KL&lt;/strong&gt;, Bertelsen S, Fagan AM, Holtzman DM, Morris JC, Kim S, Saykin AJ, De Jager PL, Albert M, Moghekar A, O'Brien R, Riemenschneider M, Petersen RC, Blennow K, Zetterberg H, Minthon L, Van Deerlin VM, Lee VM, Shaw LM, Trojanowski JQ, Schellenberg G, Haines JL, Mayeux R, Pericak-Vance MA, Farrer LA, Peskind ER, Li G, Di Narzo AF; Alzheimer’s Disease Neuroimaging Initiative (ADNI); Alzheimer Disease Genetic Consortium (ADGC), Kauwe JS, Goate AM, Cruchaga C.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Acta Neuropathol&lt;/em&gt;. 2017 May;133(5):839-856.</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;h3&gt;2016&lt;/h3&gt;</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1186/s12883-016-0742-9" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://bmcneurol.biomedcentral.com/articles/10.1186/s12883-016-0742-9"&gt;Chitinase-3-like 1 protein (CHI3L1) locus influences cerebrospinal fluid levels of YKL-40&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Deming Y, Black K, Carrell D, Cai Y, Del-Aguila JL, Fernandez MV, Budde J, Ma S, Saef B, Howells B, Bertelsen S, &lt;strong&gt;Huang KL&lt;/strong&gt;, Sutphen CL, Tarawneh R, Fagan AM, Holtzman DM, Morris JC, Goate AM, Dougherty JD, Cruchaga C.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;BMC&lt;/em&gt; &lt;em&gt; Neurology&lt;/em&gt;. 2016 Nov 10;16(1):217.</span></p>
<p class="p1"><span class="s1"><br>
&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/nature18003" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/nature18003"&gt;Proteogenomics connects somatic mutations to &lt;/a&gt;signalling&lt;a href="https://www.nature.com/articles/nature18003"&gt; in breast cancer&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Mertins P*, Mani DR*, Ruggles KV*, Gillette MA*, Clauser KR, Wang P, Wang X, Qiao JW, Cao S, Petralia F, Kawaler E, Mundt F, Krug K, Tu Z, Lei JT, Gatza ML, Wilkerson M, Perou CM, Yellapantula V, &lt;strong&gt;Huang KL&lt;/strong&gt;, Lin C, McLellan MD, Yan P, Davies SR, Townsend RR, Skates SJ, Wang J, Zhang B, Kinsinger CR, Mesri M, Rodriguez H, Ding L, Paulovich AG, Fenyö D, Ellis MJ&lt;sup&gt;#&lt;/sup&gt;, Carr SA&lt;sup&gt;#&lt;/sup&gt;; NCI CPTAC.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature&lt;/em&gt;. 2016 Jun 2;534(7605):55-62.</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/nm.4002" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/nm.4002"&gt;Systematic discovery of complex insertions and deletions in human cancers&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Ye K, Wang J, Jayasinghe R, Lameijer EW, McMichael JF, Ning J, McLellan MD, Xie M, Cao S, Yellapantula V, &lt;strong&gt;Huang KL&lt;/strong&gt;, Scott A, Foltz S, Niu B, Johnson KJ, Moed M, Slagboom PE, Chen F, Wendl MC, Ding L.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature Medicine&lt;/em&gt;. 2016 Jan;22(1):97-104.</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.4149/205_150601N306" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="http://www.elis.sk/index.php?page=shop.product_details&amp;amp;flypage=flypage.tpl&amp;amp;product_id=4569&amp;amp;category_id=85&amp;amp;option=com_virtuemart&amp;amp;vmcchk=1&amp;amp;Itemid=1"&gt;Pan-cancer methylation and expression profiling of adenocarcinomas revealed epigenetic silencing in the WNT signaling pathway&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Li J*, &lt;strong&gt;Huang KL&lt;/strong&gt;*, Zhang T, Li H, Zhao J, Wang H.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Neoplasma&lt;/em&gt;. 2016.</span></p>
<p class="p1"><span class="s1"><br>
</span></p>
<p class="p1"><span class="s1">&lt;h3&gt;2015&lt;/h3&gt;</span></p>
<p class="p1"><span class="s1">&lt;div style="float:right" data-badge-type="donut" data-badge-popover="right" data-doi="doi.org/10.1038/ncomms10086" data-hide-no-mentions="true" class="altmetric-embed"&gt;&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;span style="float:right;" class="__dimensions_badge_embed__" data-doi="10.1038/ncomms10086" data-style="small_circle" data-hide-zero-citations="true"&gt;&lt;/span&gt;</span></p>
<p class="p1"><span class="s1">&lt;a href="https://www.nature.com/articles/ncomms10086/"&gt;Patterns and functional implications of rare germline variants across 12 cancer types&lt;/a&gt;</span></p>
<p class="p1"><span class="s1">Lu C*, Xie M*, Wendl MC*, Wang J*, McLellan MD*, Leiserson MD*, &lt;strong&gt;Huang KL&lt;/strong&gt;, Wyczalkowski MA, Jayasinghe R, Banerjee T, Ning J, Tripathi P, Zhang Q, Niu B, Ye K, Schmidt HK, Fulton RS, McMichael JF, Batra P, Kandoth C, Bharadwaj M, Koboldt DC, Miller CA, Kanchi KL, Eldred JM, Larson DE, Welch JS, You M, Ozenberger BA, Govindan R, Walter MJ, Ellis MJ, Mardis ER, Graubert TA, Dipersio JF, Ley TJ, Wilson RK, Goodfellow PJ, Raphael BJ, Chen F, Johnson KJ, Parvin JD, Ding L.</span></p>
<p class="p1"><span class="s1">&lt;em&gt;Nature Communications&lt;/em&gt;. 2015 Dec 22;6:10086.</span></p>
<p class="p1"><span class="s1"><br>
<br>
</span></p>
<p class="p1"><span class="s1">&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;p class="description"&gt;&lt;/p&gt;</span></p>
<p class="p1"><span class="s1">&lt;/div&gt;</span></p>
<p class="p1"><span class="s1">&lt;/div&gt;</span></p>
</body>
</html>
