<h5>Genomics software</h5>
<a href="https://genome.cshlp.org/content/27/8/1450.full">GenomeVIP: a cloud platform for genomic variant discovery and interpretation.</a> [<a href="https://github.com/ding-lab/GenomeVIP">GitHub</a>]

<a href="https://academic.oup.com/bioinformatics/advance-article-pdf/doi/10.1093/bioinformatics/bty649/25542157/bty649.pdf">CharGer: Clinical Characterization of Germline Variants.</a> [<a href="https://github.com/ding-lab/CharGer">GitHub</a>]

<a href="https://www.worldscientific.com/doi/abs/10.1142/9789811232701_0017">AeQTL: eQTL analysis using region-based aggregation of rare variants</a>. [<a href="https://github.com/Huang-lab/AeQTL">GitHub</a>]

LOHfinder: Identify germline variants undergoing LOH, used in: <a href="https://www.cell.com/action/showPdf?pii=S0092-8674%2818%2930363-5">Pathogenic Germline Variants in 10,389 Adult Cancers.</a> [<a href="https://github.com/ding-lab/PanCanAtlasGermline/tree/master/analysis/LOH">GitHub</a>]

<h5>Proteomics software</h5>
OPPTI: <a href="https://www.nature.com/articles/s42003-021-02636-7">Pan-cancer Proteogenomic Investigations Identify Post-Transcriptional Kinase Targets</a> [<a href="https://github.com/Huang-lab/oppti">GitHub</a>]

HotPho: <a href="https://www.nature.com/articles/s41467-021-22481-w">Systematic discovery of spatially interacting phosphorylation sites and mutations in cancer</a>. [<a href="https://github.com/ding-lab/hotspot3d">GitHub for software</a>][<a href="https://github.com/ding-lab/HotPho_Analysis">GitHub for analysis scripts</a>]

KinS: <a href="https://www.mcponline.org/content/mcprot/early/2019/06/13/mcp.RA118.001243.full.pdf">Regulated phosphosignaling associated with breast cancer subtypes and druggability.</a> [<a href="https://github.com/ding-lab/phosphoproteomics">GitHub</a>]

&nbsp;

&nbsp;

&nbsp;
